Gems/LyShine/Code/Editor/EditorWindow.cpp

```diff
@@ -733,8 +733,7 @@ bool EditorWindow::SaveCanvasToXml(UiCanvasMetadata& canvasMetadata, bool forceA
             QString(),
             dir,
             "*." UICANVASEDITOR_CANVAS_EXTENSION,
-            nullptr,
-            QFileDialog::DontConfirmOverwrite);
+            nullptr);
```

Comment:
Nit: Can omit the last `nullptr` since the `selectedFilter` param has a default value of `nullptr`.